dockerclustermon/__init__.py

```diff
@@ -156,15 +161,16 @@ def process_results():
     return reduced, total, total_cpu, total_mem, used
 
 
-def run_update(username: str, host: str, no_ssh: bool, ssh_config: bool, run_as_sudo: bool):
+def run_update(username: str, host: str, no_ssh: bool, ssh_config: bool, run_as_sudo: bool, timeout: Optional[int]):
     global workers
+    results['error'] = None
```

Comment:
I did this because otherwise the errors persist across runs so error always shows instead of loading any new data. I think this is fine to clear here because if there it still an error it will get picked up again on the new run?